packages/logger/src/Logger.ts

```diff
@@ -18,6 +18,30 @@ import type {
   PowertoolLogData,
 } from './types';
 
+/**
+ *
+ * # Logger class
```

Comment:
I'd consider removing this since once rendered there's already a heading (see image below):
![image](https://user-images.githubusercontent.com/7353869/150306965-15bba9ce-b52d-435e-a5ec-eb9e3f96b7ec.png)